x-pack/legacy/plugins/ml/public/components/job_message_icon/job_message_icon.tsx

```diff
@@ -4,18 +4,19 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import React, { SFC } from 'react';
```

Comment:
`SFC` has been deprecated, we should probably be using `FC` for future work.

Reply:
Good catch - changed in 35ad48d475c32fd3c7800afabfa2c4bb5985ed99